dbt/task/run.py

```diff
@@ -2,10 +2,7 @@
 import psycopg2
 import os
 import fnmatch
-import re
-
-import sqlparse
-import networkx as nx
+from compile import Linker
```

Comment:
I'd like to avoid creating dependencies between tasks, if this needs to be used by both, factor it out somewhere outside of the `tasks` directory